cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -103,7 +103,20 @@ set(configure_command
     ${configure_command})
 
 # Pass C flags through
-set(configure_opts ${configure_opts} ${CMAKE_C_FLAGS})
+if(APPLE)
+  # Apple cannot process `-fPIC` flag argument (PR-1121)
```

Comment:
> PR-1121

It's quite hard to find relevant message, use this link: https://github.com/ruslo/hunter/pull/1109#discussion_r146360981

Reply:
Please add this link to comment.

Reply:
Still not applied.

Reply:
Just did it. Sorry missed this comment before.